samples/join-a-team-using-qr-code/nodejs/Readme.md

```diff
@@ -0,0 +1,104 @@
+---
+page_type: sample
+description: This sample demos a feature where user can join a team using QR code containing the team's id.
```

Comment:
Project folder name is still wrong "join-a-team-using-qr-code"
Please rename it as "bot-join-a-team-using-qr-code"